src/contentMetaData.js

```diff
@@ -0,0 +1,1154 @@
+            icon: 'icon-shows',
+            description: "Only have 10 minutes? These short lessons are designed to inspire you with quick tips and exercises, even if you don’t have lots of time to practice.",
+            allowableFilters: ['difficulty', 'genre', 'essential', 'theory', 'lifestyle', 'creativity'],
+            sortBy: '-published_on',
+            tabs: [
+            {
+                name: 'Lessons',
+                short_name: 'LESSONS',
+                value: '',
+            },
+            {
+                name: 'Instructors',
+                short_name: 'INSTRUCTORS',
+                is_group_by: true,
+                value: 'instructor',
+            },
+            {
+                name: 'Genres',
+                short_name: 'Genres',
+                value: 'genre',
+            },
+        ],
+    },
+    'question-and-answer': {
+        name: 'Q&A',
+            description: "Each week we go live to answer your questions. Submit your questions in advance using the button below, in the Q&A thread in the forums, or live in the community chat.",
+            allowableFilters: ['difficulty', 'genre', 'essential', 'theory'],
+            sortBy: '-published_on',
+    },
+    'workout': {
+        allowableFilters: ['difficulty', 'genre', 'topic'],
+            tabs: [
+            {
+                name: 'All',
+                short_name: 'ALL',
+                value: '',
+            },
+            {
+                name: '5 Minutes',
+                short_name: '5 MINS',
+                is_required_field: true,
+                value: 'length_in_seconds < 450',
+            },
+            {
+                name: '10 Minutes',
+                short_name: '10 MINS',
+                is_required_field: true,
+                value: 'length_in_seconds > 450 && length_in_seconds < 750',
+            },
+            {
+                name: 'Instructors',
+                short_name: 'INSTRUCTORS',
+                is_group_by: true,
+                value: 'instructor',
+            },
+        ],
+            modalText: 'Workouts are fun play-along lessons that help hone your musical skills. They cover various topics, and have multiple difficulty and duration options — so there’s always a perfect Workout for you. Just pick one, press start, and play along!',
+    },
+}
+const contentMetadata = {
+    'drumeo': {
+        'instructor': {
+            description: "Your drumming journey is unique. You need personalized coaching that helps you reach your goals. Learn from some of the best drummers in the world!",
+        },
+        'course': {
+            name: 'Courses',
+            icon: 'icon-courses',
+            description: "Tackle your next drumming goal with bite-sized courses from many of the world's best drummers.",
+            allowableFilters: ['difficulty', 'genre', 'essential', 'theory', 'lifestyle', 'creativity'],
+            sortBy: '-published_on',
+            tabs: [
```

Comment:
In general the tabs are the same for content types per each brand; there are only some situations when the tabs can be different. I think that it's a good idea to move tabs in general, if are identical.

Reply:
I've moved common course information to common, good catch on the 'course' vs 'courses'